content/en/registry/instrumentation-js-fastify.md

```diff
@@ -6,10 +6,9 @@ language: js
 tags:
   - Node.js
   - fastify
-  - fastify-plugin
-repo: https://github.com/autotelic/fastify-opentelemetry
```

Comment:
Why are we removing this instrumentation? Can't we list both?

Reply:
For both of we want to list both how do we want to go about official OpenTelemetry support vs. third party?

From what I gathered from the ticket created we want to change these over to our official repos.